Website/src/rendering/src/components/News/NewsList.tsx

```diff
@@ -11,6 +18,13 @@ type NewsListProps = ComponentProps & {
 };
 
 const NewsList = (props: NewsListProps): JSX.Element => {
+  const { sitecoreContext } = useSitecoreContext<SitecoreContextValue>();
+
+  const isPageEditing = sitecoreContext.pageState === LayoutServicePageState.Edit;
+  const hasNews = !!props.fields.items.length;
```

Comment:
This could throw if `fields`, or `items` is undefined. We could use `!!props?.fields?.items?.length` for robustness.